src/jit/lower.cpp

```diff
@@ -2963,7 +2963,7 @@ void Lowering::InsertPInvokeCallEpilog(GenTreeCall* call)
     BlockRange().InsertBefore(insertionPoint, LIR::SeqTree(comp, tree));
 
     // Pop the frame if necessary. On 32-bit targets this only happens in the method epilog; on 64-bit targets thi
```

Comment:
There is a difference between popping and deactivating a frame: the latter removes the frame from its thread's frame list, while the former leaves the frame on the list but marks it s.t. the stack walker ignores it.